### ISSUE TITLE
How to set  threshold value?

### ISSUE THREAD

**Issue by Harinisri29** [lm-sys/RouteLLM#36]
What this threshold value actually means? 
routed_model = client.route(
	prompt="What's the squareroot of 144?",
	router="bert",
	threshold=0.4066,
)
print(f"Prompt should be routed to {routed_model}")

[For 50.0% strong model calls for bert, threshold = 0.4066] what does it exactly implies? Does it mean that among 10 queries 50% will be routed to strong models?

**Comment by iojw** [comment]
We described what threshold are here: https://github.com/lm-sys/RouteLLM?tab=readme-ov-file#threshold-calibration, let me know if you have any questions!

Specifically, what it means is that for the Chatbot Arena dataset which we used to calibrate, 50% of the queries were routed to the strong model using that threshold. Depending on how similar your input queries are to the Chatbot Arena dataset, the actual % might be greater or smaller.
